codewit/client-e2e/src/e2e/cypress/e2e/1-getting-started/todo.cy.js

```diff
@@ -0,0 +1,143 @@
+/// <reference types="cypress" />
```

Comment:
It looks like this (and a lot/all of the `*.cy.js` files in sibling directories) are examples and not pertinent to the project.

Let's clean out any unnecessary files, but also add documentation within the roadmap document to include some instructions on how to navigate tests (with links to helpful examples/documentation/tutorials/etc. where appropriate).